src/quo2/components/common/notification_dot/style.cljs

```diff
@@ -4,17 +4,18 @@
 (def ^:const size 8)
 
 (defn dot-background-color
-  [customization-color theme]
-  (if customization-color
-    (colors/theme-colors
-     (colors/custom-color customization-color 50)
-     (colors/custom-color customization-color 60)
-     theme)
-    (colors/theme-colors colors/primary-50 colors/primary-60 theme)))
```

Comment:
this was defaulting to primary-50 which I don't think we want, I believe the grey color set is preferable as otherwise it is a customization color which in theory should always exist. 
If anyone knows anything about this let me know.

Cc @alwx